book/projects/index.md

```diff
@@ -1,29 +1,28 @@
 # Projects
 
-During hackweeks we invite participants to collaborate in small group project teams (usually 5-7 people). Projects provide unique opportunities for networking, advancing research and learning new things:
-
-```{image} ../img/project-purpose.png
-:alt: diagram of the purpose of hackwek project work
-:class: bg-primary mb-1
-:width: 600px
-:align: center
-```
+During hackweeks we invite participants to collaborate in small group project teams (usually 5-7 people). Projects provide unique opportunities for networking, advancing research and learning new things.
 
 We encourage you to be creative in how you design your project work! Here are some ways people have used project time in the past:
 
-
 * *focus on learning*: dive deeper into tutorial content and gain more hands on experience with new tools
 * *focus on community building*: hackweeks bring people together for the first time and projects are used to build social fabric and foster open science
 * *focus on research*: people often work to advance a research challenge using tools shared during the event. This often includes ongoing graduate and postgraduate investigations.
 * *focus on creating new tools*: hackweeks often expose general community software needs. Projects can be the catalyst for new community libraries.
 
-```{image} ../img/project-work.png
+```{image} ../img/projects-montage.png
 :alt: picture of people interacting during hackweek project work
 :class: bg-primary mb-1
-:width: 500px
+:width: 700px
 :align: center
 ```
 
-## Projects from Past Events
+## GeoSMART Hackweek Projects   
+
+The 2023 GeoSmart Hackweek will focus primarily on project work, with an emphasis on applications in Hydrology and Cyrosphere science. Project ideas will be shared in advance. Our goal is to have all participants connected to a project prior to the hackweek. 

```

Comment:
A link to the project repo template could be useful here (note that the link in project_initialization is broken)